server/install.sh

```diff
@@ -12,7 +12,7 @@
 apt-get update && apt-get install -yq yasm
 mkdir -p $GOPATH/src/github.com/minio
 cd $GOPATH/src/github.com/minio
-git clone -b deis --single-branch https://github.com/deis/minio-src.git minio
+git clone -b master --single-branch https://github.com/minio/minio.git minio
```

Comment:
`master` will probably change many times over the next few days, so the contents of the `deis/minio` image could change depending on when someone builds it. That's not good for people testing or for CI, where we need reproducible builds.

I still think using a current SHA is safer. Then when minio actually has a new release tag we can test that code and switch to that.